README.md

```diff
@@ -4,41 +4,49 @@
 
 The above description, from the [Kubernetes homepage](https://kubernetes.io/), is centered on containerized _applications_. Yet, the Kubernetes metadata, objects, and visualizations (e.g., within Dashboard) are focused on container infrastructure rather than the applications themselves.
 
-The Application CRD [(Custom Resource Definition)](https://kubernetes.io/docs/concepts/api-extension/custom-resources/#customresourcedefinitions) and [Controller](https://kubernetes.io/docs/concepts/api-extension/custom-resources/#custom-controllers) in this project aim to change that in a way that's interoperable between many supporting tools.
+Applications will provides a way for you to aggregate individual Kubernetes objects (e.g. Services, Deployments, StatefulSets, Ingresses, and CRDs), and manage them as a group. Hopefully providing UIs that allows for the aggregation and display of all the objects in the Application. 
 
-**It provides:**
+**This can be used by:**
 
-* The ability to describe an applications metadata (e.g., that an application like WordPress is running)
-* A point to connect the infrastructure, such as Deployments, to as a root object. This is useful for tying things together and even cleanup (i.e., garbage collection)
+* Dashboards that want to visualize the applications in addition to or instead of an infrastructure view
+* Application operators who want to center what they operate on applications
+* Tools, such as Helm, that center their package releases on application installations can do so in a way that's interoperable with other tools (e.g., Dashboard)
+
+**Potential Uses:**
+
+* This is useful for tying things together and even cleanup (i.e., garbage collection)
 * Information for supporting applications to help them query and understand the objects supporting an application
 * Application level health checks
 
-**This can be used by:**
+## Project Purpose
 
-* Application operators who want to center what they operate on applications
-* Tools, such as Helm, that center their package releases on application installations can do so in a way that's interoperable with other tools (e.g., Dashboard)
-* Dashboards that want to visualize the applications in addition to or instead of an infrastructure view
+* Create an Application object to store metadata about the Application
+* Create an Application object to facilitate the querying of [Kubernetes Objects](https://kubernetes.io/docs/concepts/overview/working-with-objects/kubernetes-objects/) that are associated with the Application
+* Create an Application controller to take action on the Application objects
```

Comment:
It would be helpful to clarify the purpose of the controller.  Is it a sample of a controller (say, a deployer) that acts on Application resources?  Or is it a component that should be installed into any cluster that uses the Application CRD?

Reply:
@EronWright We are taking a slow approach to introducing controller features. The controller comes automatically with the use of kubebuilder.

The first feature we are planning to introduce is in other PRs right now. It's a rolled up status of the application.